src/common/containers/landing.js

```diff
@@ -64,12 +64,12 @@ class Landing extends Component {
               Publish your software for
             </HeadingTwo>
             <div className={ `${styles.row}` }>
-              <img className={ styles.brandLogo } src={images.debian} />
-              <img className={ styles.brandLogo } src={images.opensuse} />
-              <img className={ styles.brandLogo } src={images.archlinux} />
-              <img className={ styles.brandLogo } src={images.gentoo} />
-              <img className={ styles.brandLogo } src={images.fedora} />
-              <img className={ styles.brandLogo } src={images.ubuntu} />
+              <img className={ styles.brandLogo } alt="Debian," src={images.debian} />
+              <img className={ styles.brandLogo } alt="openSUSE," src={images.opensuse} />
+              <img className={ styles.brandLogo } alt="Arch Linux," src={images.archlinux} />
+              <img className={ styles.brandLogo } alt="Gentoo," src={images.gentoo} />
+              <img className={ styles.brandLogo } alt="Fedora," src={images.fedora} />
+              <img className={ styles.brandLogo } alt="Ubuntu." src={images.ubuntu} />
```

Comment:
Can we say more than just the logo company, we need to explain that his is the logo of that particular company.